src/søknad/steg/3-barnadine/LeggTilBarn.tsx

```diff
@@ -70,28 +66,29 @@ const LeggTilBarn: React.FC<Props> = ({ settÅpenModal, id }) => {
     const fødselsnummer =
       barnDato && personnummer ? formatDateFnr(barnDato) + personnummer : '';
 
-    const barn = {
-      fnr: fødselsnummer,
-      personnummer: personnummer,
-      alder: differenceInYears(dagensDato, barnDato),
-      navn: navn,
-      fødselsdato: formatDate(barnDato),
-      harSammeAdresse: boHosDeg === 'ja',
-      født: født,
-      lagtTil: true,
-      id: hentUid(),
-    };
-
-    const nyttBarn = settLabelOgVerdi(barn, standardLabelsBarn);
+    if (født) {
+      const nyttBarn: IBarn = hentNyttBarn(
+        fødselsnummer,
+        personnummer,
+        barnDato,
+        navn,
+        boHosDeg,
+        født,
+        intl
+      );
```

Comment:
Jeg har tweaket litt på metoden vi bruker for å hente og sette` {label, verdi}` for når man legger til nytt barn. Av den grunn at det er litt vanskelig å sette {spørsmålid, svarid} med metoden i dag. 🤔 
I` settLabelOgVerdi( objekt, labels )`, så tar du inn et objekt med "vanilla" `datatyper {string, boolean}` og setter på label tatt inn som et argument. Så hvis labelslista man tar inn ikke er i riktig rekkefølge, så vil labelsene også bli satt feil. 😋 Også har jeg bare behov for å kun utvide` barn.født` feltet til `ISpørsmålBooleanFelt {spørsmålid, svarid, label, verdi}`. Så har løst dette med en metode `hentFeltObjekt() `som returnerer label og verdi som støtter intl-oversetting, og en metode` hentNyttBarn()` som setter barnet (slik som du har satt det opp, formaterer noen av verdiene før de settes først) med `hentFeltObjekt() `for `{label, verdi} `og legger på `{spmid og svarid}` for `født`. 

Innspill eller tanker? :)

Reply:
Oh, nice! Fint å bruke `hentFeltObjekt()` til å håndtere labelen og å dra ut `hentNyttBarn` i en metode :)